wgpu-core/src/resource.rs

```diff
@@ -1552,6 +1552,9 @@ pub struct TextureViewDescriptor<'a> {
     /// - For 2D textures it must be one of `D2`, `D2Array`, `Cube`, or `CubeArray`.
     /// - For 3D textures it must be `D3`.
     pub dimension: Option<wgt::TextureViewDimension>,
+    /// The allowed usage(s) for the texture view. Must be a subset of the usage flags of the texture.
+    /// If 0, defaults to the full set of usage flags of the texture.
+    pub usage: wgt::TextureUsages,
```

Comment:
The way we express this for other members is to make them optional, where None = inherit. I think we probably should do that here too.

Reply:
I think in this case, the better api would be to be self-consistent, rather than consistent with the spec here - it's what users of a rusty api would come to expect and is self-explanatory for people coming from the JS spec.